### ISSUE TITLE
[Proposal] Add "evidence" of talk quality

### ISSUE THREAD

**Issue by mattstauffer** [tighten/symposium#99]
Allow for tracking feedback on a talk. Tweets, maybe others?

```
adamwathan [10:23 AM] 
hey good idea for symposium, storing feedback to use as part of future submissions
if i could log in to my talk and hit like "add evidence!" and paste a twitter link
that would be slick
```

**Comment by rdohms** [comment]
So this matches na Idea i had and i'm starting to think it belongs in Symposium and not outside of it, but it could go either way.

Anyway, the part that is relevant here, is adding to a talk a link to a joindin talk.
With these id's you could generate a graph of evolution, get a few quotes and even list all conferences where the user spoke at.

i did something similar: http://aggregatein.azurewebsites.net/talk/annotating-with-annotations

**Comment by mattstauffer** [comment]
@rdohms OH! Clever! I like that graph idea.

**Comment by mattstauffer** [comment]
Thinking so far:

A "submission" is a link between a talk and a conference. (will need renaming.)

It can progress for "applied" to "rejected" or to "accepted" or to "given" (or "past" or whatever.)

It is the unique entity to represent "When I gave my Empathy talk at Laracon EU 2015" vs "When I gave my Empathy talk at the White House that one time". So each Submission would be the place to add feedback.

Currently thinking: 
- Submission can have a Joind In Talk URL, from which we pull reviews like @rdohms mentioned above
- Submission hasMany Twitter reviews, each of which is a tweet URL
- Submission hasMany plain reviews, which are just text (and a name? and an attribution??)
- Submission hasMany URL reviews?  or should it just be twitter vs. plain?

**Comment by rdohms** [comment]
I would argue against the "text" review and rather invest in getting people to do that on joind.in, so i would really make it very low priority.

I'm a bit put off by the "applied, rejected" states, for one we can't automate this and why would i want to manually add this before actually presenting the talk? The only states that r "to me" really make sense, would be "accepted or upcoming" and "done" which would be the same state but automated and time sensitive. Thus i could say "this is where i will be at" which is very cool and "where i have been"... not sure i want to say "this is where i was rejected, bastards" 

I would suggest the term "performance" if we are focusing only on positives not on rejections.

**Comment by mattstauffer** [comment]
- The "text" review was only for if you have gotten evidence for conference that don't use Joind.in. Not the primary way to do it, for sure.
- Applied, rejected, etc.. primarily so that you can mark that you "applied" to a particular conference with a particular talk. I've gotten acceptance emails from a conference and couldn't even remember which talk I submitted. Additionally, it's so you can remember what you submitted to a conference when you submit to the same conference the next year. Does that make sense? Oh, also: the rejected state is only for internal knowing, not for public showing, just read your last sentence.

Performance would be good if it's only positives; trying to think of something that encompasses:
- I submitted talk and haven't heard back yet
- I submitted the talk and was rejected
- I submitted the talk and was accepted; haven't given yet
- I have given the talk

**Comment by andrewmile** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add evidence of talk quality](https://trello.com/c/3xGjRYm5/86-add-evidence-of-talk-quality)

**Comment by andrewmile** [comment]
Addressed in #392
